### PR TITLE
Initalize postgre sql db

### PR DESCRIPTION
**Pull request by shohamroditimemphis**

### CODE REVIEW

File: db/db.go [hunk 1]
```diff
@@ -156,6 +285,409 @@ func EditSystemKey(key string, value string) error {
...
+		cancelfunc()
+		return DbPostgreSQLInstance{}, err
+	}
+
+	err = dbPostgreSQL.Ping(ctx)
+	if err != nil {
+		cancelfunc()
+		return DbPostgreSQLInstance{}, err
+	}
+	l.Noticef("Established connection with the PostgreSQL DB")
```
Comment (idanasulinmemphis) [inline db/db.go:645]: Established connection with the meta-data storage

File: db/db.go [hunk 2]
```diff
@@ -156,6 +285,409 @@ func EditSystemKey(key string, value string) error {
...
+	conn, err := dbPostgreSQL.Acquire(ctx)
+	if err != nil {
+		cancelfunc()
+		return DbPostgreSQLInstance{}, err
+	}
+	dbPostgre := DbPostgreSQLInstance{Ctx: ctx, Cancel: cancelfunc, Client: dbPostgreSQL, Conn: conn}
+	err = createTablesInDb(dbPostgre)
```
Comment (idanasulinmemphis) [inline db/db.go:652]: createTables

File: db/db.go [hunk 3]
```diff
@@ -156,6 +285,409 @@ func EditSystemKey(key string, value string) error {
...
+	// if err != nil {
+	// 	return DbPostgreSQLInstance{}, err
+	// }
+
+	// err = dropRowInTable(dbPostgre)
+	// if err != nil {
+	// 	return DbPostgreSQLInstance{}, err
+	// }
+
+	// err = JoinTable(dbPostgre)
```
Comment (idanasulinmemphis) [inline db/db.go:683]: you can remove these comments

File: db/db.go [hunk 4]
```diff
@@ -121,6 +139,117 @@ func Close(dbi DbInstance, l logger) {
...
+
+	return nil
+}
+
+func InsertToTable(dbPostgre DbPostgreSQLInstance) error {
+	ctx, cancelfunc := context.WithTimeout(context.Background(), dbOperationTimeout*time.Second)
+	defer cancelfunc()
+	conn := dbPostgre.Conn
+	defer conn.Release()
+	_, err := conn.Conn().Prepare(ctx, "insert into", `INSERT INTO tags (name, color, users, stations, schemas)
```
Comment (idanasulinmemphis) [inline db/db.go:177]: always put the query in a separate string variable

**Review by idanasulinmemphis — APPROVED** [review]

**Comment by Mazeian** [comment]
Guys help create my payment. Php document on my loan management System
project

**Pull request merged**